provisioning_virtual_machines_and_hosts/_topics/fulfilling_requests.md

```diff
@@ -175,27 +175,35 @@ Grayed out items reflect values that are set in the class schema. These values c
 ##### Customizing Provisioning States
 
 The steps followed when provisioning a virtual machine or cloud instance are completed based on instances from the
-menu:DOMAIN\[Cloud|Infrastructure \> VM \> Provisioning \> StateMachines \> VMProvision\_VM\] class. Depending on your environment you can remove, change, or add steps to the provisioning process. For example, if you are not integrating with IPAM or a CMDB, then you can remove
-those execution steps. ![6281](../images/6281.png)
+menu:DOMAIN\[Cloud|Infrastructure \> VM \> Provisioning \> StateMachines \> VMProvision\_VM\] class.
 
 ##### Editing the Default State Instance
 
-1.  Navigate to menu:Automate\[Explorer\].
+1.  Navigate to menu:Automation \> Embedded Automate \> Explorer.
 
-2.  From the accordion menu, click menu:DOMAIN\[Cloud \> VM \> Provisioning \> StateMachines \> VMProvision\_VM\].
+2.  Click **Datastore** accordion. Select **Configuration** > Add a New Domain. You must fill the Name and Description values and click **Add** to create a custom 
+    domain.
 
-    **Note:**
+3.  From the accordion menu, navigate to ManageIQ \> Cloud \> VM \>Provisioning \> StateMachines \> VMProvision_VM.
```

Comment:
We should avoid `ManageIQ` in the docs, as it affects productization

```suggestion
3.  From the accordion menu, navigate to Cloud \> VM \>Provisioning \> StateMachines \> VMProvision_VM.
```

Reply:
Done